### PR TITLE
Updated py-currentscape version.

### PR DESCRIPTION
**Pull request by AurelienJaquier**

### CODE REVIEW

File: deploy/packages/bbp-packages.yaml [hunk 1]
```diff
@@ -131,7 +131,7 @@ packages:
       - py-bluepy@0.14.15
       - py-bluepymm@0.7.65 ^neuron%intel@19.0.4
       - py-bluepyopt@1.9.27 ^neuron%intel@19.0.4
-      - py-currentscape@0.0.6
+      - py-currentscape@0.0.9
```
Comment (matz-e) [inline deploy/packages/bbp-packages.yaml:134]: This is obsolete. Is there an entry for `py-currentscape` in `deploy/environments/applications.yaml`? If that does not have a version number, it should automatically deploy the "latest" stable version (0.0.9 in this case)

**Review by matz-e — APPROVED** [review]
Good to merge from my side.

File: deploy/packages/bbp-packages.yaml (hunk 1 above)
Comment (AurelienJaquier) [reply deploy/packages/bbp-packages.yaml:134]: There is an entry in deploy/environments/applications.yaml. Should I remove the version number here?

Comment (matz-e) [reply deploy/packages/bbp-packages.yaml:134]: No need for that. I'll remove these files in a future PR, they are now not used any longer

Comment (AurelienJaquier) [reply deploy/packages/bbp-packages.yaml:134]: Alright, then. Thank you.

**Pull request merged**